game/templates/game/landing.html

```
@@ -1,1992 +1,1420 @@
-          <div class="about-box">
+    </div>
+  </div>
+</section>
+
+<section id="about" class="about-pin-section">
+  <div class="about-pin-sticky">
+    <h2 class="section-title">About Us</h2>
+
+    <div class="about-box">
+      <button
+        id="aboutPausePlayBtn"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)